src/cclib/io/ccio.py

```diff
@@ -363,6 +364,8 @@ def _determine_output_format(outputtype, outputdest):
             outputclass = cmlwriter.CML
         elif outputtype.lower() == 'xyz':
             outputclass = xyzwriter.XYZ
+        elif outputtype.lower() == 'molden':
```

Comment:
Would it make sense to use a dictionary for this mapping instead of all the conditions?